_vendor/github.com/chef/chef-workstation/docs-chef-io/content/workstation/install_workstation.md

```diff
@@ -47,7 +47,7 @@ Supported Host Operating Systems:
 </tr>
 <tr class="odd">
 <td>Red Hat Enterprise Linux / CentOS</td>
-<td>7.x, 8.x</td>
+<td>7.x, 8.x, 9.x</td>
```

Comment:
Can you undo this change. This should be made from the workstation repo. See chef/chef-workstation#3117

Reply:
Done ✅